inspirehep/modules/theme/ext.py

```diff
@@ -59,6 +59,7 @@ def init_app(self, app, assets=None, **kwargs):
 
         # Add Jinja2 extensions.
         app.jinja_env.add_extension('jinja2.ext.do')
+        app.jinja_env.autoescape = False
```

Comment:
Have a look if there is a better method for escaping for your needs instead of globally disabling http://flask.pocoo.org/docs/0.10/templating/#controlling-autoescaping